src/agent/block_store_services/block_store_fs.js

```diff
@@ -43,7 +43,7 @@ class BlockStoreFs extends BlockStoreBase {
         this.xattr_enabled = true;
 
         this.fs_context = {
-            disable_ctime_check: config.BLOCK_STORE_FS_TMFS_ENABLED
+            do_ctime_check: !config.BLOCK_STORE_FS_TMFS_ENABLED
```

Comment:
@tangledbytes @guymguym I kept only this ctime check, do we want a ctime check here?